packages/cojson/src/sync.ts

```diff
@@ -115,6 +116,12 @@ export class SyncManager {
       | undefined;
   } = {};
 
+  peersCounter = metrics.getMeter("cojson").createUpDownCounter("jazz.peers", {
+    description: "Amount of connected peers",
+    valueType: ValueType.INT,
+    unit: "peer",
+  });
```

Comment:
suggestion: what about creating a metrics adapter?

Something like:
```ts
class Metrics {
    peersCounter = metrics.getMeter("cojson").createUpDownCounter("jazz.peers", {
      description: "Amount of connected peers",
      valueType: ValueType.INT,
      unit: "peer",
    });
    
    trackAddPeer(peerRole: Peer['role']) {
       this.peersCounter.add(1, { role: peer.role });
    }
    
    trackRemovePeer(peerRole: Peer['role']) {
       this.peersCounter.add(-1, { role: peer.role });
    }
}

export const metrics = new Metrics();
```

Should make it easier to write tests and makes the tracking calls cleaner.

Reply:
saw this a bit too late, but i think it's worth exploring. I'll experiment with it and follow up if that's ok